test/common/index.js

```diff
@@ -429,8 +435,14 @@ function runCallChecks(exitCode) {
   if (failed.length) process.exit(1);
 }
 
-
-exports.mustCall = function(fn, expected) {
+/**
+ * @function mustCall
+ * @param fn "The code that must be called
+ * @param {?number|string} expected "Expected times to be called, or '+' for more then 0. Default = 1"
+ * @returns {Function}
+ * @export
+ */
+exports.mustCall = function mustCall(fn, expected) {
```

Comment:
the JSDocs?
It's usefull for people with IDEs that can parse these (VSCode / Webstorm)